lib/shared/log/tasks.js

```diff
@@ -13,7 +13,23 @@ function isObject(val) {
   return typeof val === 'object' && !Array.isArray(val);
 }
 
+function wrapGetTask(getTask) {
+  return function(name) {
+    return createTask(getTask(name));
+  };
+}
+
+function createTask(task) {
+  var unwrapped = task.unwrap ? task.unwrap() : {};
```

Comment:
Which do you think best to check if `task.description` is not empty (or specified)?

```js
 1. if (task.description) ...
 2. if (task.description != null) ...
 3. if (typeof task.description === 'string') ...
 4. if ('description' in task) ...
```